frontend/packages/console-dynamic-plugin-sdk/src/runtime/plugin-manifest.ts

```diff
@@ -38,7 +38,7 @@ export const fetchPluginManifest = async (baseURL: string) => {
 
   const response: Response = await coFetch(url, { method: 'GET' });
   const manifest = (await response.json()) as ConsolePluginManifestJSON;
-
-  (await validatePluginManifestSchema(manifest, url)).report();
+  // Disabled to allow injecting custom extension points at runtime
+  // (await validatePluginManifestSchema(manifest, url)).report();
```

Comment:
We still want to valid the manifest, right? We should just tolerate unknown extensions?

cc @christianvogt @vojtechszocs

Reply:
@spadgett `report(false)` should allow us to validate without throwing an error.

Reply:
We still want to throw errors on other problems. We just want to ignore extensions that we don't know about.

Reply:
Right, we should apply the validation only to Console core extensions, excluding any (plugin-specific) ones - basically tolerate unknown extensions, as Sam wrote above.